Source/WebCore/rendering/MarkedText.h

```diff
@@ -68,24 +69,53 @@ struct MarkedText {
         Frontmost
     };
 
-    unsigned startOffset;
-    unsigned endOffset;
-    Type type;
-    const RenderedDocumentMarker* marker { nullptr };
-    AtomString highlightName { };
+    MarkedText(unsigned startOffset, unsigned endOffset, Type type, const RenderedDocumentMarker* marker = nullptr, AtomString highlightName = { }, int priority = 0) : startOffset(startOffset), endOffset(endOffset), type(type), marker(marker), highlightName(highlightName), priority(priority) { };
+    MarkedText(WTF::HashTableDeletedValueType) : startOffset(std::numeric_limits<unsigned>::max()) { };
+    MarkedText() { };
```

Comment:
`MarkedText() = default;` would make it clear to the compiler this is still the default constructor implementation, not a custom one.